### PR TITLE
Tiny refactor

### PR DESCRIPTION
**Pull request by fisker**
- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/pr-XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: src/language-js/printer-estree.js [hunk 1]
```diff
@@ -3927,25 +3927,24 @@ function printJestEachTemplateLiteral(node, expressions, options) {
...
-    table
-      .filter(row => !row.hasLineBreak)
-      .forEach(row => {
-        row.cells.forEach((cell, index) => {
-          maxColumnWidths[index] = Math.max(
-            maxColumnWidths[index],
-            getStringWidth(cell)
-          );
-        });
+    const maxColumnWidths = Array.from({ length: maxColumnCount }).fill(0);
```
Comment (armano2) [inline src/language-js/printer-estree.js]:
```suggestion
    const maxColumnWidths = new Array(maxColumnCount).fill(0);
```

Comment (fisker) [reply src/language-js/printer-estree.js]: No , thanks, I  think this is exactly why es has `Array.from` `Array.of`.

Comment (armano2) [reply src/language-js/printer-estree.js]: https://2ality.com/2018/12/creating-arrays.html#cheat-sheet%3A-creating-arrays
https://2ality.com/2018/12/creating-arrays.html#recommended-patterns

Comment (fisker) [reply src/language-js/printer-estree.js]: Personally I always use `Array.from({length})` to generate array of this length, I think it's best practice.

Another PR for eslint also use it https://github.com/eslint/eslint/pull/12608/files

Comment (armano2) [reply src/language-js/printer-estree.js]: they are using it in different way, they are using callback to prefill values

```suggestion
    const maxColumnWidths = Array.from({ length: maxColumnCount }, () => 0);
```

Comment (armano2) [reply src/language-js/printer-estree.js]: they are using it in different way, they are using callback to prefill values

```suggestion
    const maxColumnWidths = Array.from({ length: maxColumnCount }, () => 0);
```

Comment (armano2) [reply src/language-js/printer-estree.js]: they are using it in different way, they are using callback to prefill values

```js
    const maxColumnWidths = Array.from({ length: maxColumnCount }, () => 0);
```

Comment (fisker) [reply src/language-js/printer-estree.js]: I wrote them differently on purpose. There is no difference in this case

Comment (fisker) [reply src/language-js/printer-estree.js]: There is no they, still me in that PR.

**Pull request merged**